bin/burden

```diff
@@ -1173,18 +1173,30 @@ aws_specific_os_version()
 retrieve_spot_from_config()
 {
 	if [[ $gl_use_spot -eq 1 ]]; then
-		if [[ -e ${gl_test_def_dir}/spot_price.cfg ]]; then
-			spot_string=`grep $1 ${gl_test_def_dir}/spot_price.cfg`
-			if [[ $? -eq 0 ]]; then
-				gl_spot_price=`echo ${spot_string} | cut -d':' -f2`
-				gl_spot_increment=`echo ${spot_string} | cut -d':' -f3`
-				gl_spot_cap=`echo ${spot_string} | cut -d':' -f4`
-			else
-				cleanup_and_exit "Error: no spot information present for $1" 1
+		spot_string=`grep $1 ${gl_test_def_dir}/spot_price.cfg`
+		if [[ $gl_system_type == "aws" ]]; then
+			#
+			# For AWS obtain the possible starting price.
+			#
+			gl_spot_price=`aws ec2 describe-spot-price-history --start-time=$(date +%s) --product-descriptions="Linux/UNIX" --query 'SpotPriceHistory[*].{az:AvailabilityZone, price:SpotPrice}' --instance-types ${1} | awk '{print $2}' | sort -n | tail -1`
```

Comment:
Does Linux/UNIX match RHEL?  Since RHEL adds a slight extra fee I'm wondering how that all works out.  Maybe a non-issue?

Reply:
Non issue, it will not be much of a difference, KISS approach.